float32 posX
float32 posY

float32 sizeX
float32 sizeY

float32 rotation